_posts/zh_CN/pages/2016-01-13-segwit-support.md

```diff
@@ -0,0 +1,34 @@
+---
+type: pages
+layout: page
+lang: zh_CN
+name: 2016-01-13-segwit-support
+id: zh_cn-2016-01-13-segwit-support
+title: 隔离见证支持
+permalink: /zh_CN/segwit_adoption/
+version: 1
+share: false
+---
+
+如下是宣布将支持隔离见证的公司和项目清单。
+
+最新! (隔离见证比特币测试网服务):
+
+|Name|Planned|Ready|Notes|Depends|
+|----|-------|-----|-----|-------|
+{% include pages/segwit_support.md %} 
+
+添加公司或服务, 请使用 ACK ticket [#30] 加公司／服务名称。
```

Comment:
Looks like the hyperlink is gone,  should be

`[#30](https://github.com/bitcoin-core/bitcoincore.org/pull/30)` off the top of my head